### ISSUE TITLE
[Web] [Accessibility][11yMAS][Screen Readers][MAS4.2.10][HighImpact][Narrator focus is not in synchronization with Tab focus in Filter by product combo box.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#692]
**User Experience:**
User who rely on AT will not get information about “Controls in Filter by product combo box” if narrator focus is not in sync with tab focus.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools Used: Narrator
**Attachments:**
[MAS4.2.10_Focus is not in sync.zip](https://github.com/OfficeDev/script-lab/files/1445688/MAS4.2.10_Focus.is.not.in.sync.zip)

**Repro Steps:**

1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “Filter by product “ combo box and press Enter on it. 
3. Navigate through the Items that are present in combo box using Down arrow key.
4. Verify Narrator focus is in sync with Tab focus and verify screen reader is announcing those List items inside the combo box.

 **Current Behavior:** 
Narrator focus is in sync with Tab focus so that screen reader is not announcing those List items inside the Filter by product combo box.

**Expected Behavior:**
Narrator focus should be in sync with Tab focus so that screen reader will be able to announce those items and user will get the information about those list item which are present in Filter by product combo box.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.2.10
